StripeConnect/StripeConnect/Source/Helpers/StripeConnectBundleLocator.swift

```diff
@@ -1,15 +0,0 @@
-//
-//  StripeConnectBundleLocator.swift
-//  StripeConnect
-//
-//  Created by Chris Mays on 7/31/24.
-//
-
-import Foundation
-@_spi(STP) import StripeCore
-
-final class StripeConnectBundleLocator: BundleLocatorProtocol {
```

Comment:
To support SPM, we would have needed to add `static let spmResourcesBundle = Bundle.module` ([Example](https://github.com/stripe/stripe-ios/blob/5541e9b3/Stripe/StripeiOS/Source/StripeBundleLocator.swift#L16-L18)). However, SPM only generates this if there's a non-empty file in the `Resources` directory, which we don't yet have.